specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-11-01-preview/DynamicSummaries.json

```diff
@@ -0,0 +1,398 @@
+          "type": "string"
+        },
+        "tactics": {
+          "items": {
+            "$ref": "./common/AlertTypes.json#/definitions/AttackTactic"
+          },
+          "description": "A list of relevant MITRE attacks",
+          "type": "array"
+        },
+        "techniques": {
+          "description": "The list of techniques for summary",
+          "items": {
+            "description": "Related Technique",
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "relationName": {
+          "description": "The relation name for the summary",
+          "type": "string"
+        },
+        "relationId": {
+          "description": "The relation Id (GUID) for the summary",
+          "type": "string"
+        },
+        "summaryDescription": {
+          "description": "The description for the summary",
+          "type": "string"
+        },
+        "rawContent": {
+          "description": "The raw content that represents to summary items to create.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of the summary",
+          "enum": [
+            "dynamic-summary",
+            "dynamic-summary-item"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "type"
+          }
+        },
+        "tenantId": {
+          "description": "The tenantId where the summary belongs to",
+          "type": "string"
+        },
+        "summaryStatus": {
+          "description": "The status of the Summary : Active, Deleted",
+          "enum": [
+            "Active",
+            "Deleted"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "summaryStatus"
+          }
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the Dynamic Summary resource."
+        }
+      },
+      "required": [
+        "summaryName"
+      ],
+      "type": "object"
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "The current provisioning state.",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "InProgress"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    }
+  },
+  "parameters": {
+    "SummaryId": {
+      "description": "Summary Id",
+      "in": "path",
+      "name": "summaryId",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
is it in any particular format? arm-id? uuid?

Reply:
the SummaryId value is a GUID